firmware/network/border_router/include/border_router.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @{
+ * @ingroup     network
+ * @file        border_router.h
+ * @brief       this module content all functions of Border Router
+ * @author      RocioRojas <rociorojas391@gmail.com>
+ *
+ */
+
+#ifndef BORDER_ROUTER_H
+#define BORDER_ROUTER_H
+
+#include "net/gnrc.h"
+#include "net/netdev.h"
+#include "net/netif.h"
+#include "net/ipv6/addr.h"
+#include "net/gnrc/netif.h"
+#include "net/netdev/ieee802154.h"
+
```

Comment:
Do you need all these headers in this .h? I think that could only need `net/ipv6/addr.h